networking/network_security/AdminNetworkPolicy/ovn-k-anp-recommended-practices.adoc

```diff
@@ -0,0 +1,11 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="ovn-k-anp-recommended-practices"]
+= Best practices for AdminNetworkPolicy
+include::_attributes/common-attributes.adoc[]
+:context: ovn-k-anp-banp-metrics
+
+toc::[]
+
+This section provides recommended best practices for the `AdminNetworkPolicy` and `BaselineAdminNetworkPolicy` resources.
```

Comment:
Not a major issue. IMO, `recommended` can sound a bit redundant because best practices are generally what's recommended. Feel free to take a call here.

```suggestion
This section provides best practices for the `AdminNetworkPolicy` and `BaselineAdminNetworkPolicy` resources.
```